integration/src/test/resources/scatter-gather-test.xml

```diff
@@ -17,8 +17,6 @@
             <set-payload value="banana"/>
```

Comment:
Removed the config file. I could see that the class that used it was ScatterGatherRouterTestCase but it does not exist anymore